templates/login.html

```diff
@@ -0,0 +1,93 @@
+{% extends "page.html" %}
+
+{% block login_widget %}
+{% endblock %}
+
+{% block main %}
+
+{% block login %}
+
+<div style="font-size:25px;font-weight:bold; margin-bottom: 150px"  align="left" >
+    <p  align="left" > notices:</p>
```

Comment:
I think this should be aligned by "Center" . And maybe a `h1` tag should be awesome to do the font size thing.